asyncnsq/tcp/reader.py

```diff
@@ -92,7 +93,7 @@ async def connect(self):
                     loop=self._loop)
                 await self.prepare_conn(conn)
                 self._connections[conn.id] = conn
-            self._rdy_control.add_connections(self._connections)
+            self._rdy_control.add_connections(self._connections) #do we need to close conn in rdy_control?
```

Comment:
rdy_control only for manipulate rdy state

Reply:
Sorry, I forgot the clean this comment. That was a note for myself when I was reading the code.

Reply:
I removed the comment in the latest commit.